loginFragments/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <application>
```

Comment:
This looks too simplified? Where are all the other meta property tags

Reply:
Removed all of this and collapsed this tag since its in the app module